x-pack/filebeat/module/cisco/asa/test/dap_records-expected.json

```diff
@@ -0,0 +1,18 @@
+{
+    "source": {
+      "ip": "1.2.3.4"
+    },
+    "user": {
+      "email": "firstname.lastname@domain.net"
+    },
+    "cisco": {
+      "connection_type": "AnyConnect",
+      "dap_records": [
+        "dap_1",
+        "dap_2"
+      ],
+      "asa": {
+        "message_id": "734001"
+      }
+    }
+}
```

Comment:
This file should be named `dap_records.log-expected.json`, and the content must be generated by the system tests. Otherwise is ignored.

To generate the content of this file, you must run the module's tests in generate mode (first fix the issues with the pipeline I pointed out in other comments):

1. Create a virtualenv:
```bash
cd {beats_repo}/x-pack/filebeat
virtualenv -p path_to_python_3 venv
. venv/bin/activate
pip install -r ../../libbeat/tests/system/requirements.txt
```

2. Then run the cisco/asa tests in generate mode:
```bash
touch module/cisco/asa/test/dap_records.log-expected.json
GENERATE=1 MODULES_PATH=$PWD/module INTEGRATION_TESTS=1 TESTING_FILEBEAT_MODULES=cisco TESTING_FILEBEAT_FILESETS=asa nosetests -v -s tests/system/test_xpack_modules.py
```

3. Run the cisco tests in testing mode and it should pass:
```bash
 MODULES_PATH=$PWD/module INTEGRATION_TESTS=1 TESTING_FILEBEAT_MODULES=cisco nosetests -v -s tests/system/test_xpack_modules.py
```